sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -19,7 +21,7 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder)
                 throw new ArgumentNullException(nameof(builder));
             }
 
-            builder.AddEventHubs(p => {});
+            builder.AddEventHubs(ConfigureOptions);
```

Comment:
What if someone calls the other overload?

There is already configuration parsing code in the `AddEventHubs(this IWebJobsBuilder builder, Action<EventHubOptions> configure)` overload, let's chain the new code there.

Reply:
Yeah, I wasn't sure about this, but this is how it works in the currently released library. I assumed that if someone called the lower level method, they are assuming responsibility for how the configuration should work. /cc @sidkri

Reply:
Yeah, this is the default behavior.  If someone calls "AddEventHubs(IWebJobsBuilder, Action<EventHubOptions>)" I'm assuming they want to have full control over the behavior including changing default behavior.

Reply:
I think it's a bit unintuitive that trying to change some unrelated setting would stop loading the starting point configuration especially considering that the rest of the settings are being bound unconditionally.

Reply:
Agreed, at the very least it feels inconsistent that some settings are still defaulted, and some are not.